CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # PayPal Android SDK Release Notes
 
 ## unreleased
+* Add [Gradle Nexus Publish Plugin](https://github.com/gradle-nexus/publish-plugin) to release process
```

Comment:
We should be able to remove this line in the CHANGELOG since it isn't a merchant facing change.